n2v/models/n2v_standard.py

```diff
@@ -88,6 +89,11 @@ def __init__(self, config, name=None, basedir='.'):
         self.keras_model = self._build()
         if config is None:
             self._find_and_load_weights()
+        
+        
+        print("config.structN2Vmask", config.structN2Vmask)
```

Comment:
Does this print statement serve a debugging purpose? Would it be possible to remove it or is it necessary?